Project 7 - Bankist/index.html

```diff
@@ -0,0 +1,361 @@
+                      class="user-avatar"
+                      alt="amazon"
+                    />
+                  </div>
+                  <div class="transaction__label">
+                    <p class="transaction__correspondents">Sanjay</p>
+                    <p class="transaction__date">10 - 07 - 2023</p>
+                  </div>
+                </div>
+                <div class="transaction__amount">
+                  <p class="transaction__value">+ $ 1000</p>
+                  <p class="transaction__type">credit</p>
+                </div>
+              </div>
+              <div class="transaction__list">
+                <div class="transaction__user">
+                  <div class="transaction__icon">
+                    <img
+                      src="images/flipkart.png"
+                      class="user-avatar"
+                      alt="amazon"
+                    />
+                  </div>
+                  <div class="transaction__label">
+                    <p class="transaction__correspondents">flipkart</p>
+                    <p class="transaction__date">10 - 07 - 2023</p>
+                  </div>
+                </div>
+                <div class="transaction__amount">
+                  <p class="transaction__value">- $ 1000</p>
+                  <p class="transaction__type">debit</p>
+                </div>
+              </div>
+            </div>
+          </section>
+        </section>
+      </main>
+
+      <aside class="sidebar">
+        <div class="card sidebar__card">
+          <div class="card__title">Transfer money</div>
+          <form class="form card__body" id="form-transfer">
+            <div>
+              <input
+                type="text"
+                list="receiver-name-list"
+                id="receiver-name"
+                required
+                class="input__group"
+                placeholder="Choose account"
+              />
+              <datalist id="receiver-name-list"> </datalist>
+            </div>
+            <div>
+              <input
+                type="number"
+                id="amount-transfer"
+                class="input__group"
+                placeholder="enter amount"
+                required
+              />
+            </div>
+            <div class="btn-container">
+              <button class="btn btn--primary" type="submit">Send</button>
+            </div>
+          </form>
+        </div>
+        <div class="card sidebar__card">
+          <div class="card__title">Request loan</div>
+          <form class="form card__body" id="form-loan">
+            <div>
+              <select name="" id="loan-type" class="input__group select_input">
+                <option value="" selected>Select</option>
+                <option value="home">Home</option>
+                <option value="education">Education</option>
+                <option value="car">Car</option>
+              </select>
+              <!-- <select
```

Comment:
@sknepali Let's remove this dead code if not needed

Reply:
Done